Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -866,6 +858,67 @@ def get_alert_by_id(client, alert_id, token, url):
         raise DemistoException(f"[get_alert_by_id] Error during HTTP request: {str(e)}")
 
 
+def update_alert_data_command(client: Client, url: str, token: str, args: Dict[str, Any]) -> CommandResults:
+    """
+    Update alert data (status/severity) on Cyble Vision platform for one or more alerts.
+    """
+    demisto.debug(f"[update_alert_data_command] Raw args: {args}")
+
+    ids = argToList(args.get("ids"))
+    statuses = argToList(args.get("status")) if args.get("status") else []
+    severities = argToList(args.get("severity")) if args.get("severity") else []
+
+    demisto.debug(f"[update_alert_data_command] Parsed ids: {ids}")
+    demisto.debug(f"[update_alert_data_command] Parsed statuses: {statuses}")
+    demisto.debug(f"[update_alert_data_command] Parsed severities: {severities}")
```

Comment:
```suggestion
    demisto.debug(f"Parsed ids: {ids}")
    demisto.debug(f"Parsed statuses: {statuses}")
    demisto.debug(f"Parsed severities: {severities}")
```
no need to have the command name in the log, the logs will show that they belong to this command

Reply:
It makes it easier for us to search the logs.